syntax = "proto2";

import "States.proto";
package pds;

message podCommand {
    optional ControlsInterfaceStates controlsInterfaceState = 1;
    optional PodStates manualPodState = 2;
    optional BrakeNodeStates manualBrakeNodeState = 3;
    optional MotorStates manualMotorState = 4;
    optional LvdcNodeStates manualLvdcNodeState = 5;
    optional bool automaticStateTransitions = 6;
}